src/frontend/komponenter/Fagsak/Saksoversikt/Behandling.tsx

```diff
@@ -0,0 +1,119 @@
+
+import {
+    behandlingsresultater,
+    behandlingsstatuser,
+    BehandlingStatus,
+    behandlingstyper,
+    behandlingÅrsak,
+} from '../../../typer/behandling';
+import type { IBehandlingstema } from '../../../typer/behandlingstema';
+import {
+    hentKategorierHvisVisningBehandling,
+    tilBehandlingstema,
+} from '../../../typer/behandlingstema';
+import type { IMinimalFagsak } from '../../../typer/fagsak';
+import type { ITilbakekrevingsbehandling } from '../../../typer/tilbakekrevingsbehandling';
+import {
+    Tilbakekrevingsbehandlingstype,
+    tilbakekrevingstyper,
+} from '../../../typer/tilbakekrevingsbehandling';
+import { datoformat, formaterIsoDato } from '../../../utils/formatter';
+import type { VisningBehandling } from './visningBehandling';
+
+interface IBehandlingshistorikkProps {
+    minimalFagsak: IMinimalFagsak;
+    behandling: VisningBehandling | ITilbakekrevingsbehandling;
+}
+
+const erTilbakekrevingsbetaling = (behandling: VisningBehandling | ITilbakekrevingsbehandling) => {
+    return tilbakekrevingstyper.some(type => type === behandling.type);
+};
+
+const lagLenkePåType = (
+    fagsakId: number,
+    behandling: VisningBehandling | ITilbakekrevingsbehandling
+): ReactNode =>
+    behandling.status === BehandlingStatus.AVSLUTTET ? (
+        behandlingstyper[behandling.type].navn
+    ) : erTilbakekrevingsbetaling(behandling) ? (
+        <Link
+            href={`/redirect/familie-tilbake/fagsystem/BA/fagsak/${fagsakId}/behandling/${behandling.behandlingId}`}
+            onMouseDown={e => e.preventDefault()}
+            target="_blank"
+        >
+            <span>{behandlingstyper[behandling.type].navn}</span>
+            <ExternalLink />
+        </Link>
+    ) : (
+        <Link href={`/fagsak/${fagsakId}/${behandling.behandlingId}`}>
+            {behandlingstyper[behandling.type].navn}
+        </Link>
+    );
+
+const lagLenkePåResultat = (
+    minimalFagsak: IMinimalFagsak,
+    behandling: VisningBehandling | ITilbakekrevingsbehandling
+): ReactNode => {
+    if (!behandling.resultat) {
+        return '-';
+    }
+    if (erTilbakekrevingsbetaling(behandling)) {
+        return (
+            <Link
+                href={`/redirect/familie-tilbake/fagsystem/BA/fagsak/${minimalFagsak.id}/behandling/${behandling.behandlingId}`}
+                onMouseDown={e => e.preventDefault()}
+                target="_blank"
+            >
+                <span>{behandlingsresultater[behandling.resultat]}</span>
+                <ExternalLink />
+            </Link>
+        );
+    } else {
+        if (behandling.status === BehandlingStatus.AVSLUTTET) {
+            return (
+                <Link href={`/fagsak/${minimalFagsak.id}/${behandling.behandlingId}`}>
+                    {behandling ? behandlingsresultater[behandling.resultat] : '-'}
+                </Link>
+            );
+        } else {
+            return behandlingsresultater[behandling.resultat];
+        }
+    }
```

Comment:
Er ikke dette det samme som
```suggestion
    } else if (behandling.status === BehandlingStatus.AVSLUTTET) {
            return (
                <Link href={`/fagsak/${minimalFagsak.id}/${behandling.behandlingId}`}>
                    {behandling ? behandlingsresultater[behandling.resultat] : '-'}
                </Link>
            );
     else {
        return behandlingsresultater[behandling.resultat];
    }
```
?